### ISSUE TITLE
Bookmarks translator fails to unescape HTML character references in title names.

### ISSUE THREAD

**Issue by ckush** [zotero/translators#1485]
When importing bookmarks from a .html file, if the title of a bookmark has some characters as numeric references, such as &#39; or &amp;,  the Bookmarks.js translator fails to convert it to the original character.

It find that changing...

`openItem.title = title;`

in Bookmarks.js to...

`openItem.title = ZU.unescapeHTML(title);`

solves the problem. 

I realise this is a minor change but I'm new to programming and would love it if I'm allowed to go through the process of submitting this change myself, for educational purposes.

Below is a .html bookmarks file you can test this with. Note that collection names are imported fine, the problem is only in bookmark titles.

```
<!DOCTYPE NETSCAPE-Bookmark-file-1>
<!-- This is an automatically generated file.
     It will be read and overwritten.
     DO NOT EDIT! -->
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
<TITLE>Bookm &amp; ardsd ks</TITLE>
<H1>Book &amp; marks</H1>
    <DT><H3 ADD_DATE="1502137359" LAST_MODIFIED="1502137366">Progra &amp; mming</H3>
    <DL><p>
        <DT><A HREF="http://code.google.com/edu/languages/google-python-class/introduction.html" ADD_DATE="1329052843">Python Introduction - Google&#39;s Python Class - Google Code</A>
    </DL>
</DL>
```

**Comment by adam3smith** [comment]
That sounds right -- and yes, absolutely, I'd be happy to take a pull request from you that fixes this. Let me know if you need any directions.

**Comment by ckush** [comment]
OK I'm going to make a pull request. I think I understand what to do but if I make a mistake please excuse this noob.

**Comment by zuphilip** [comment]
Solved by #1496.
